### PR TITLE
Distance/Similarity

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#349 #350
#### What does this implement/fix? Explain your changes.
- `distance` method added
- 30 new distance/similarity added
	1. AMPLE
	2. Anderberg's D
	3. Andres & Marzo's Delta
	4. Baroni-Urbani & Buser I
	5. Baroni-Urbani & Buser II
	6. Batagelj & Bren
	7. Baulieu I
	8. Baulieu II
	9. Baulieu III
	10. Baulieu IV
	11. Baulieu V
	12. Baulieu VI
	13. Baulieu VII
	14. Baulieu VIII
	15. Baulieu IX
	16. Baulieu X
	17. Baulieu XI
	18. Baulieu XII
	19. Baulieu XIII
	20. Baulieu XIV
	21. Baulieu XV
	22. Benini I
	23. Benini II
	24. Canberra
	25. Clement
	26. Consonni & Todeschini I
	27. Consonni & Todeschini II
	28. Consonni & Todeschini III
	29. Consonni & Todeschini IV
	30. Consonni & Todeschini V

### CODE REVIEW

File: Document/Distance.ipynb [hunk 1]
```diff
@@ -0,0 +1,1652 @@
...
+    "from pycm import ConfusionMatrix, DistanceType"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cm = ConfusionMatrix(matrix = {0: {0: 3, 1: 0, 2: 0}, 1: {0: 0, 1: 1, 2: 2}, 2: {0: 2, 1: 1, 2: 3}})"
```
Comment (sadrasabouri) [inline Document/Distance.ipynb:39]: Can we use this, instead?
`cm = ConfusionMatrix(matrix={0: {0: 3, 1: 0, 2: 0}, 1: {0: 0, 1: 1, 2: 2}, 2: {0: 2, 1: 1, 2: 3}})`

File: Document/Distance.ipynb [hunk 2]
```diff
@@ -0,0 +1,1652 @@
...
+       "{0: 0.6, 1: 0.3, 2: 0.17142857142857143}"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cm.distance(metric = DistanceType.AMPLE)"
```
Comment (sadrasabouri) [inline Document/Distance.ipynb:91]: Can we use this, instead?
`cm.distance(metric=DistanceType.AMPLE)`

Comment (sepandhaghighi) [reply Document/Distance.ipynb]: fixed in https://github.com/sepandhaghighi/pycm/pull/475/commits/d97768a38e85497bbaa2056e9a351b85f090f3de

File: Document/Distance.ipynb (hunk 1 above)
Comment (sepandhaghighi) [reply Document/Distance.ipynb]: fixed in https://github.com/sepandhaghighi/pycm/pull/475/commits/d97768a38e85497bbaa2056e9a351b85f090f3de

**Review by sadrasabouri — APPROVED** [review]
Great.

**Pull request merged**